### ISSUE TITLE
Allow fixed length importer to have option to ignore quote handling completely

### ISSUE THREAD

**Issue by orothad** [OpenRefine/OpenRefine#4101]
Found a related issue around the quote handling problems I was trying to find from previous data exploration.  And came upon this one where I **had** to use FIXED LENGTH importer and set it manually to 1000 for column length (a quick workaround for the Line-based importer bug that I'll describe later in another issue.)

There were a few lines that appeared to have wrapped around into a new row ??? ...still analyzing those handful to see where the real issue may lie, but here's some preview and I'll update this feature in a bit.

I think the importer is probably OK, but just might need more special options to deal with this.
Might also end up being related to some of what we talked about before in #724 

I used `smartSplit()` here just to quickly see what was going on potentially with surrounding lines and where the data got messy all of a sudden at row 2912896 or if the parser was somehow too strict or lax.

```
<html><body>
<!--StartFragment-->

2912896. | netherlands,"",owner-of-praktijk-laren,mental health care,linkedin.com/company/owner-of-praktijk-laren,"","praktijk laren | Error: smartSplit error: Un-terminated quoted field at end of CSV line
-- | -- | --
2912897. | www.ip-praktijk.nl","",1-10,ip-praktijk.nl | Error: smartSplit error: Un-terminated quoted field at end of CSV line

<!--EndFragment-->
</body>
</html>
```

![image](https://user-images.githubusercontent.com/88521629/129265306-e26b9697-3945-42f3-98ac-76aa6507e425.png)

### Proposed solution
Allow fixed length importer to have an option to ignore quote handling completely and only worry about `\n` or `\cr\lf` at end of line.

### Additional context
Below is the custom output from only those rows that had an error on the `smartSplit()` for analysis.
You can see that there's definitely a \n somewhere that caused OpenRefine's Fixed Length importer to create the new row, but still connected data of the original row that had mixed quoting on certain fields apparently.  So truly messy data. :-)
Yeah, I think this might be very similar in nature or identical to #724 
But on this issue we can probably call this a feature request to give better support when rows have that "mixed structure" of cells themselves which might throw off certain importers.
Letting the user determine what the real cell delimited strategy should be I think could go a long way towards helping the problem.

In fact, I could envision a new visual interface area for all the importers themselves to actually deal with the problems of
* "choose your **cell/field** delimited strategy from the options below".
* "choose your **line** delimited strategy from the options below".

and in choose the options, pre-parsing and error warnings might appear to the user as suggested in #724 
Just other tools do.

```
Column 1
"united kingdom,"""",restorex-limited,venture capital & private equity,linkedin.com/company/restorex-limited,duns,""restorex "
"cuisine pro sa"",the scottish borders,1-10,restorexltd.co.uk"
"netherlands,"""",owner-of-praktijk-laren,mental health care,linkedin.com/company/owner-of-praktijk-laren,"""",""praktijk laren"
"www.ip-praktijk.nl"","""",1-10,ip-praktijk.nl"
"united kingdom,2011,fillcon,packaging and containers,linkedin.com/company/fillcon,clevedon,""fillcon"
"sachet manufacturers"",north somerset,1-10,fillconuk.com"
"south africa,"""",specialised-power-solutions,"""",linkedin.com/company/specialised-power-solutions,meyerton,""specialised power"
"electrical engineers & manufacturers"",gauteng,1-10,"""""
"united states,"""",hospitality-contractors-inc,hospitality,linkedin.com/company/hospitality-contractors-inc,plant city,""chef steve "
trainer assessor
```

**Comment by orothad** [comment]
@wetneb Thoughts on above?  I think I've explained the issue and probable quick solution (giving Fixed Length that option), and long solution (rethinking how we deal with cell/line delimiting strategies presented to the user on each importer).

**Comment by wetneb** [comment]
I haven't looked into it but at first I wonder why the fixed length importer would deal with quotes in a special way. I suspect it treats them like any other character. Perhaps the line-based importer would be more suitable to your use case?

**Comment by thadguidry** [comment]
@wetneb I'm scratching my head this morning...I've tried line-based importer previously on the same large csv and it ate up over 20gb memory.  I tried again this morning and it's working and only 5gb used in the heap.  I suspect that perhaps it was something on my end. I don't know.

So no need for this feature, as I had a hunch, since I've used line-based so many times.  I'm surprised that I had to resort to the Fixed-width fields importer.

Regardless, we don't need this feature, and indeed a line-based importer should not have to worry about quotes or much of anything except a stream of characters and separating on each line delimiter such as `\n` or `\r\n`.  Which many times in the past it has shown it works, even with millions of rows in a dataset, such as my case.

So, I will close this feature issue now.
